validator/client/runner.go

```diff
@@ -40,21 +43,27 @@ func run(ctx context.Context, v Validator) {
 		log.Fatalf("Could not wait for validator activation: %v", err)
 	}
 	if err := v.UpdateAssignments(ctx, params.BeaconConfig().GenesisSlot); err != nil {
-		log.WithField("error", err).Error("Failed to update assignments")
+		handleAssignmentError(err, params.BeaconConfig().GenesisSlot)
```

Comment:
Why do you need to pass this constant as a function argument? You could fetch it from the method itself

Reply:
Because a few lines down we also call handleAssignmentError but the slot can be different. For the first call, it'll always be genesis slot but in the future, it might be slot 400, etc.